src/main/java/com/movie_collection/HelloApplication.java

```diff
@@ -19,16 +16,17 @@ public void start(Stage stage) throws IOException {
         stage.setScene(scene);
         stage.show();
 
-        try {
-            System.out.println(cm.getConnection().isClosed());
-        } catch (SQLException e) {
-            throw new RuntimeException(e);
-        }
+
     }
 
     public static void main(String[] args) {
         launch();
 
-
+//        MovieTest mt = new MovieTest();
```

Comment:
yep also not needed in the development

Reply:
resolved in 
https://github.com/TomassSimko/Private-Movie-Collection/pull/7/commits/c327b4d35a561959bc104c192001071ff3d61114